### DISCUSSION TITLE
Add: Remote branches are on local, not on the remote repository

### DISCUSSION THREAD

**Discussion by ooker777** [pcottle/learnGitBranching#967] [Ideas]
In lesson `remote2` and so on, it takes me a while and several rereads to realize that the remote branch is on local, not on the remote repository. Therefore, I suggest we to add this note on the first popup of the `remote2` lesson:

> To be clear: Remote branches are on your _local_ repository, not on the remote repository

**Comment by pcottle** [comment]
Yeah that sounds good! want to attempt the pull request to change this?

**Reply by ooker777** [reply]
let me get to that lesson first :P
